iam/snippets/src/test/java/DenyIT.java

```diff
@@ -37,7 +37,6 @@
 public class DenyIT {
 
   private static final String PROJECT_ID = System.getenv("IAM_PROJECT_ID");
-  private static final String GOOGLE_APPLICATION_CREDENTIALS = System.getenv("IAM_CREDENTIALS");
```

Comment:
Okay, I finally realized what this is for. The sample is explicitly pulling the [service account ADC](https://cloud.google.com/docs/authentication/application-default-credentials#GAC) from the environment for ... some reason. Is the sample creating a new service account and assigning it to the env var?